packages/ed25519-signature-2018/package.json

```diff
@@ -45,6 +45,7 @@
   "dependencies": {
     "@transmute/ed25519-key-pair": "0.7.0-unstable.2",
     "@transmute/jose-ld": "^0.7.0-unstable.42",
+    "@transmute/linked-data-proof": "^0.7.0-unstable.42",
```

Comment:
This should be in dev dependencies, it is only used in jest test

Reply:
Moving this now!